ftl/core/editing.ftl

```diff
@@ -68,3 +68,13 @@ editing-close-html-tags = Auto-close HTML tags
 ## You don't need to translate these strings, as they will be replaced with different ones soon.
 
 editing-html-editor = HTML Editor
+
+## Image Occlusion
```

Comment:
The string above is deprecated; it would be nice if that was left at the bottom of the file, with the other strings being added above.